### DISCUSSION TITLE
macOs Ventura Removes Option to Duplicate Terminal

### DISCUSSION THREAD

**Discussion by osmanmerdan** [biostars/biostar-handbook-issues#247] [Q&A]
Hi everyone,
 Apparently with the new macOs Ventura we can't duplicate the terminall.app to use duplicated one with rosetta. That was the way suggested by the biostar-handbook. I wonder is there a simple walk-around for this issue?

**Comment by ialbert** [comment]
that is an unexpected development, 

I believe that at this time, you can now use a non-rosetta terminal as well - so in the end it simplifies the tasks.

Can you run through this with a non-rosetta terminal and verify that it is indeed working?

**Reply by osmanmerdan** [reply]
I have run a simple variant calling script, and it looks fine. There are no errors or something. Thanks for your rapid response and support.

**Reply by ialbert** [reply]
Excellent news! I have removed the Rosetta related instructions since those are now outdated, the terminal will automatically invoke rosetta as necessary.
